app/main.py

```diff
@@ -1 +1,6 @@
-# write your code here
+def copy_file(command: str) -> None:
+    cp_command, current_file, future_file = command.split()
+    if current_file == future_file and cp_command == "cp":
```

Comment:
if current_file == future_file but command will not equal to cp -> we will do copy
Strange logic